packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -463,3 +463,27 @@ export const getProposalRequestObject = ({
 
     return request;
 };
+
+export const getPayoutInfo = (proposal_info: ReturnType<typeof getProposalInfo>) => {
```

Comment:
Here we are extracting payout  and max payout from proposal. In case if proposal contains error, we do not receive receive this fields. Instead we are extracting payout and max payout from error text with reg exp :( 
Need future improvement: ask BE team to add fields in error details.